powerhome/playbook
# WIP - [PLAY-35] Add Spline example to Bar Graph docs

#### Screens

##### Rails Example
<img width="997" alt="rails_example" src="https://user-images.githubusercontent.com/17501514/142052009-c4421745-28fb-4523-b00b-22494675a93b.png">

##### Rails Code Snippet
<img width="984" alt="Rails_snippet" src="https://user-images.githubusercontent.com/17501514/142051835-2f07483a-4de3-4c9b-8341-8b1a20125083.png">

##### React Example
<img width="981" alt="React_example" src="https://user-images.githubusercontent.com/17501514/142052132-b4a32329-fe7e-474e-a817-9be5e0c71e73.png">

##### React Code Snippet
<img width="983" alt="React_snippet" src="https://user-images.githubusercontent.com/17501514/142052222-b40ed163-96f8-4ec8-adf2-98826bf77cd1.png">

#### Breaking Changes

No. This PR adds an example for an already existing, but undocumented, feature of bar graphs.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-35

#### How to test this

- Verify the example appears in the rails bar graph examples
- Verify the example appears in the react bar graph examples

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
